### ISSUE TITLE
Week 13 - 회의 기록

### ISSUE THREAD

**Issue by 4thlee** [sangmandu/2022-DCC#36]
# [날짜]
| 10월 31일(월) 

# [시간 조정]
- 화 : 21시
- 수 : 21시
- 목 : 21시

# 이태훈
- 진행중 : augmentation 좋은 것 찾기 
- RandomAffine 에서 이미지 짤림이 성능에 악영향 줄 수 있음

# 박민영 & 조소연
- 진행중 : 옵티마이저 실험
- 중복 이미지 제거
	- 제거할 대상(리스트) 외에 key도 추가로 return 해보기
- radam
- madgrad

# 전상민
- 진행중 : Dup 실험 , Cutmix and Crop 실험
- Dup 실험
	- Dup 1.0 vs Dup 0.9 : Dup 0.9가 성능이 더 좋음
	- Dup 0.91 이 성능이 가장 좋았음
- pretrain 모델
	- 기존 모델 사용에 따른 감점 우려가 발생, 이에 따라 NAS 찾아봄
	- NAS : 에폭 돌때마다 , 최적 layer 를 찾아줌

# 박종원
- 진행중 : Dup 실험, Loss 함수 실험
- Dup 실험
	- Dup 1.0 이 가장 좋았음 
	- 상민님과 다른 이유
		- 스케줄러 차이 or 학습 데이터양 차이
- Loss 함수 실험
	- best : cross_entropy
- 오버샘플링 실험
	- best : 각 클래스 모두 1000씩 + aug 
	- 결론 
		- 오버샘플링할수록 성능이 상승한다
		- 오버샘플링 후에 augmentation하면 성능이 상승한다
		- aug를 잘하면 더 성능이 높아질 것이다
- 앞으로 PPT 작업 시작 or K-fold 구현
